src/schema/course/utils.ts

```diff
@@ -0,0 +1,31 @@
+import queryBuilder from '../../utils/queryBuilder';
+import { CourseParams } from './types';
```

Comment:
You can also use generated types `import {QueryCourseListArgs} from "../../types/types"`